src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/CancelRequests/CancelRequestItem.tsx

```diff
@@ -0,0 +1,80 @@
+import clsx from 'clsx';
```

Comment:
Is there a way to reuse code from similar components? Composition via props could be used for elements specific to this type of action, e.g. the penalise badges.

Reply:
@jakubcolony Can we do this in other issue and refactor all request items?